Tasks/GradlePreview/task.json

```diff
@@ -101,39 +93,33 @@
             "label": "CodeCoverage Tool",
             "required": false,
             "groupName": "codeCoverage",
-            "defaultValue": "JaCoCo",
-            "helpMarkDown": "Select the tool with which summary files have been generated. Only Jacoco support available currently.",
+            "defaultValue": "NoCoverage",
+            "helpMarkDown": "Select the tool for evaluating code coverage. Only Jacoco support available currently.",
             "options": {
+                "NoCoverage": "No Coverage",
                 "JaCoCo": "JaCoCo"
             }
         },
         {
-            "name": "summaryFileLocation",
+            "name": "classfilesDirectory",
             "type": "string",
-            "label": "Summary File",
-            "defaultValue": "$(Build.Repository.LocalPath)\\build\\reports\\jacoco\\test\\jacocoTestReport.xml",
+            "label": "Classfiles Directory",
```

Comment:
Class Files Directory. Everywhere make sure ant, gradle and maven have consistency in UI